index.bs

```diff
@@ -271,7 +374,40 @@ interface FileSystemFileHandle : FileSystemHandle {
 };
 </xmp>
 
-A {{FileSystemFileHandle}}'s associated [=FileSystemHandle/entry=] must be a [=file entry=].
+A {{FileSystemFileHandle}}'s associated [=FileSystemHandle/locator=]'s
+[=file system locator/kind=] is {{FileSystemHandleKind/"file"}}.
```

Comment:
You could make this a note. (Unless text requires something I usually make it either a note or an example. If you think this is a better you should feel free to leave it as-is though. It's definitely an improvement over what was here before.)

Reply:
Seems reasonable to make this a note